DAL/Data Transfer Objects/UserDetailsDTO.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace DAL.Data_Transfer_Objects
 {
-    class UserDetailsDTO
+    public class UserDetailsDTO
```

Comment:
Volgens de PR is Users en UserDetails ongebruikt, waarom blijven ze dan bestaan.

Reply:
Removed